src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_WellKnownAttributes.cs

```diff
@@ -9130,17 +9130,43 @@ public void M()
 }
 ";
 
-            var comp = CompileAndVerify(source);
+            var source_skip = @"
+[module: System.Runtime.CompilerServices.SkipLocalsInitAttribute]
 
-            Assert.Null(comp.HasLocalsInit("<>f__AnonymousType0<<Value>j__TPar>.GetHashCode"));
-            Assert.Null(comp.HasLocalsInit("<>f__AnonymousType0<<Value>j__TPar>..ctor"));
-            Assert.True(comp.HasLocalsInit("<>f__AnonymousType0<<Value>j__TPar>.Equals"));
-            Assert.True(comp.HasLocalsInit("<>f__AnonymousType0<<Value>j__TPar>.ToString"));
-            Assert.Null(comp.HasLocalsInit("<>f__AnonymousType0<<Value>j__TPar>.Value.get"));
+namespace System.Runtime.CompilerServices
+{
+    public class SkipLocalsInitAttribute : System.Attribute
+    {
+    }
+}
+
+public class C
+{
+    public void M()
+    {
+        var anon = new { Value = 1 };
+    }
+}
+";
+
+            var comp_init = CompileAndVerify(source_init);
+            var comp_skip = CompileAndVerify(source_skip, verify: Verification.Fails);
+
+            Assert.Null(comp_init.HasLocalsInit("<>f__AnonymousType0<<Value>j__TPar>.GetHashCode"));
+            Assert.Null(comp_init.HasLocalsInit("<>f__AnonymousType0<<Value>j__TPar>..ctor"));
```

Comment:
The "Has" prefix almost always indicates that the return type is bool. If you want to return the attribute instance I would call this method `GetLocalsInitAttributeOpt`

Reply:
I'm returning a nullable bool. True if `locals init (` is present in IL, false if `locals (` is present, null if none is present.